src/sentry/incidents/subscription_processor.py

```diff
@@ -552,7 +552,9 @@ def trigger_alert_threshold(
         """
         self.trigger_alert_counts[trigger.id] += 1
 
-        if options.get("metric_alerts.rate_limit"):
+        if features.has(
+            "organizations:metric-alert-rate-limiting", self.subscription.project.organization
```

Comment:
should i add a check that the project still exists? i noticed another feature flag in this file had a try/except to check that the project being referenced exists.

Reply:
No, the project should exist if the subscription exists - we'd cascade delete the subscription otherwise.